src/classes/Trades.ts

```diff
@@ -738,7 +738,7 @@ export = class Trades {
 
             offer.data('finishTimestamp', finishTimestamp);
 
-            const processTime = finishTimestamp - offer.data('handleTimeStamp');
+            const processTime = finishTimestamp - offer.data('handleTimestamp');
```

Comment:
Nice haha

Reply:
Yea, I am surprised no one realized that. It's frustrating to see it always say unknown ms so I take a look and found this.